index.html

```diff
@@ -27,5 +27,121 @@ <h3><a href="#" class="action-text">LET’S CONNECT</a></h3>
             <li><a href="https://github.com/mudasiri"><i class="fa-brands fa-medium"></i></a></li>
         </ul>
     </section>
+    <section class="works">
+        <div class="grid-container">
+            <div class="works-flex-container">
+                <img class="portfolio-item-img" src="img/tonic.png" alt="tonic Website">
+                <h1 class="portfolio-title">Tonic</h1>
+                <p class="history" id="bold-history">CANOPY</p>
+                <p class="history"><span><i class="fa-solid fa-circle fa-2xs"></i></span> Backend Dev</p>
+                <p class="history"><span><i class="fa-solid fa-circle fa-2xs"></i></span> 2015</p>
+                <p class="project-desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <ul id="portfolio-technologies">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>JAVASCRIPT</li>
+                </ul>
+                <button class="see-demo-btn" type="button">See Project</button>
+            </div>
+
+            <div class="works-flex-container">
+                <img class="portfolio-item-img" src="img/portfolio2.png" alt="tonic Website">
+                <h1 class="portfolio-title">Multi-Post Stories</h1>
```

Comment:
- [ OPTIONAL ] I recommend add h2 or h3 tag instead of h1 tag because you should use just one h1 tag per page that's very important for SEO and to have semantic page 🔥💯🚀